Dockerfile

```
@@ -17,14 +17,24 @@ RUN \
   rm util-linux-${utillinux_version}.tar.xz hashes
 
 # Build script for SSM session logging
-WORKDIR /opt/build/util-linux-${utillinux_version}
+WORKDIR $HOME/build/util-linux-${utillinux_version}
+RUN ./autogen.sh
 RUN \
-  ./autogen.sh && ./configure \
-        --disable-all-programs \
-        --enable-scriptutils \
+  ./configure \
+      --disable-makeinstall-chown \
+      --disable-nls \
+      --disable-rpath \
+      --prefix=/opt/util-linux \
+      --without-audit \
+      --without-python \
+      --without-readline \
+      --without-systemd \
+      --without-udev \
+      --without-utempter \
     || { cat config.log; exit 1; }
+RUN make -j`nproc` lscpu
 RUN make -j`nproc` script
```

nit: could potentially combine these:
```suggestion
RUN make -j`nproc` lscpu script
```